lib/Resque/Job.php

```diff
@@ -243,6 +243,11 @@ public function fail($exception)
 				$this->queue
 			);
 		}
+		
+		if(!empty($this->payload['id'])) {
+			Resque_Job_PID::del($this->payload['id']);
```

Comment:
With namespaces this should now be `Resque\Job\PID`. Please add a use statement at the top and use `PID` here.
The filename of this file changed from `Job.php` to `JobHandler.php`